### DISCUSSION TITLE
【灌水】家人们谁懂啊，青衣歪了

### DISCUSSION THREAD

**Discussion by liuyaozhu** [share121/inter-knot#2094] [其他]
难受啊

**Comment by Mine268** [comment]
摸

**Comment by BSNHD** [comment]
加油肝一肝，说不定还有希望<img src="https://upload-bbs.mihoyo.com/upload/2022/11/24/78c0c40f7cc89facd10ff38b3385d918_3614720584318019134.png" alt="纳西妲-沉思中" width="75" >
